src/components/CreateChatroom.js

```diff
@@ -0,0 +1,72 @@
+import React, {useEffect, useState} from 'react';
+import {BaseUrl} from "../Constants";
+import axios from "axios";
+
+function CreateChatroom(props) {
+    const [users, setUsers] = useState([])
+
+useEffect(() => {
+      let config = {
+            method: 'get',
+            maxBodyLength: Infinity,
+            url: BaseUrl+"chat/users/",
+            headers: {
+                'Authorization': 'token d3e47a7b4f84ce3d4c7ea397ce118e801a0f0588'
+            }
+        };
+
+        axios.request(config)
+            .then((response) => {
+                setUsers((response.data));
+            })
+            .catch((error) => {
+                console.log(error);
+            });
+},[])
+
+    const handleAdd = () => {
+        let name = document.getElementById("name").value
+        let created_by = document.getElementById("created_by").value
+        let members = document.getElementById("members").value
+        let data = JSON.stringify({
+            "name": name,
+            "created_by": created_by,
+            "members": [members]
+        })
+
+        let config = {
+            method: 'post',
+            maxBodyLength: Infinity,
+            url: BaseUrl+"chat/chatroom/",
+            headers: {
+                'Authorization': 'token d3e47a7b4f84ce3d4c7ea397ce118e801a0f0588',
+                'Content-Type': 'application/json'
+            },
+            data: data
+        };
+        axios.request(config)
+            .then((response) => {
+                console.log(JSON.stringify(response.data));
+            })
+            .catch((error) => {
+                console.log(error);
```

Comment:
In case of error, we need to show the error to the user. You can use an error state to set the errors and render it on the form.